packages/happo-target-react-native/package.json

```diff
@@ -9,13 +9,16 @@
   "scripts": {
     "build:runner:ios": "./scripts/build-runner-ios.sh",
     "build:runner:android": "./scripts/build-runner-android.sh",
+    "build:runner:windows": "./scripts/build-runner-windows.sh",
     "build:server": "../../node_modules/.bin/babel src -d lib --ignore __tests__",
     "build": "npm run build:runner:ios && npm run build:runner:android && npm run build:server",
     "watch": "npm run build:server -- --watch"
   },
   "devDependencies": {
     "react-native": "^0.40.0",
-    "react-native-view-shot": "^1.5.1"
+    "react-native-view-shot": "^1.5.1",
+    "react-native-windows": "^0.43.0-rc.0",
```

Comment:
Interesting. I'm tempted to agree this should be an optionalDependency.  We could even create `happo-target-react-native-windows`, which would include this dependency.  It may be a bit more work to modularize the code properly then.